setup.py

```diff
@@ -30,5 +30,15 @@
     ],
     keywords='Exoplanets imaging high-contrast coronagraphy',
     install_requires=[
-        'astropy', 'configobj', 'flake8', 'jupyter', 'matplotlib', 'numpy', 'pytest', 'scikit-image', 'scipy'
-    ])
+        "astropy",
+        "matplotlib",
+        "configobj",
+        "IPython",
```

Comment:
You did indeed reorder, but not alphabetically? 🤔 Also, I think `ipython` should have no caps?

Reply:
ok, it was an attempt to do it alphabetically :-D

Also, you seem to be right about the caps. Why did the pytest installation worked ?

Reply:
Ah xD I have no clue. Maybe it's makes it all lower caps automatically. Still, worth it to keep things clean here.